src/foam/nanos/auth/UserLocatorService.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.INTERFACE({
+  package: 'foam.nanos.auth',
+  name: 'UserLocatorService',
+
+  methods: [
+    {
+      name: 'getUser',
+      type: 'foam.nanos.auth.User',
+      args: [
```

Comment:
args: 'Context x, String identifier, String password'

Reply:
yes, got it. Thank you!